frontend/main.py

```diff
@@ -252,6 +260,17 @@ def delete_correction(queue_name, correction_id, version):
     get_correction_table().delete_item(Key={'queue': queue_name, 'id': str(
         correction_id)}, ExpressionAttributeValues={':v1': version}, ConditionExpression='version = :v1')
 
+def update_correction_queue(queue_name, correction_id, version, updated_queue_name):
+    try:
+        #it is not possible to update an attribute that forms the Key of an item thus we need to copy/delete/put in DynamoDB.
+        correction = read_correction(queue_name, correction_id, version)
+        delete_correction(queue_name, correction_id, version)
+        correction['version'] = 0
+        correction['queue'] = updated_queue_name
```

Comment:
you need to fix the lastAssigned as well. Let us reduce the code duplication and use a helper method in this and skip method

Reply:
factored out a helper method to reset_correction(correction, reset_fields)